app/src/main/java/com/globant/pokemontcgapp/adapter/PokemonSupertypesAdapter.kt

```diff
@@ -0,0 +1,39 @@
+package com.globant.pokemontcgapp.adapter
```

Comment:
is necessary this new adapter? can't you reuse the ones that you made for the types?